.gitignore

```diff
@@ -157,4 +157,4 @@ cython_debug/
 #  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
```

Comment:
JetBrains advises here not including .idea in the GITIGNORE - I'm not sure why because we obviously aren't meant to commit our own .idea folders. I think @rothoma2 will know more on this.

Reply:
Sure, please let me know can revert. I use PyCharm and it was unnecessarily always in staging area.

Reply:
I use it too, and for my own projects always gitignore it. Just better double check with Robert - he'll probably know why JetBrains discourage it.

Reply:
I also dont know, but lets keep the .idea on the GITIGNORE. If you want to use a specific IDE, keep the file locally and dont put it on the repo.